README.md

```diff
@@ -91,18 +91,16 @@ Check the below video for a quick demo of XC3.
     ```
     cd xc3/infrastructure
     bash init.sh
+    chmod 400 "keypair.pem"
```

Comment:
Add this command before sudo ssh command not here

Reply:
Feedback addressed.